plugin/src/main/java/io/jenkins/plugins/casc/core/GlobalNodePropertiesConfigurator.java

```diff
@@ -0,0 +1,77 @@
+package io.jenkins.plugins.casc.core;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.slaves.EnvironmentVariablesNodeProperty.Entry;
+import io.jenkins.plugins.casc.Attribute;
+import io.jenkins.plugins.casc.BaseConfigurator;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorException;
+import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
+import io.jenkins.plugins.casc.model.CNode;
+import io.jenkins.plugins.casc.model.Mapping;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Extension
+public class GlobalNodePropertiesConfigurator extends BaseConfigurator<EnvironmentVariablesNodeProperty> {
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "globalNodeProperties";
+    }
+
+    @Override
+    protected EnvironmentVariablesNodeProperty instance(Mapping mapping, ConfigurationContext context)
+            throws ConfiguratorException {
+        List<Entry> vars = getVarsAsList(mapping);
+        return new EnvironmentVariablesNodeProperty(vars);
+    }
+
+    @Override
+    public Class<EnvironmentVariablesNodeProperty> getTarget() {
+        return EnvironmentVariablesNodeProperty.class;
+    }
+
+    @NonNull
+    @Override
+    public Set<Attribute<EnvironmentVariablesNodeProperty, ?>> describe() {
+        Set<Attribute<EnvironmentVariablesNodeProperty, ?>> attrs = super.describe();
+        attrs.add(new MultivaluedAttribute<EnvironmentVariablesNodeProperty, EnvironmentVariablesNodeProperty.Entry>(
```

Comment:
Is this needed?

Reply:
Would extending `HeteroDescribableConfigurator` work instead?

Reply:
I didn't test that, seems more complex than this approach.
I can do a test and check if we would be simplifying things

Reply:
Sorry, my fault, not Hetero, but DataBoundConfigurator.
Just changed it to use it, slightly simpler.

Reply:
Looks, much simpler and more like what I expected, thanks for sorting.